src/open_producten/locaties/tests/api/test_contact.py

```diff
@@ -12,6 +12,7 @@
 
 
 class TestContact(BaseApiTestCase):
+    path = reverse("contact-list")
```

Comment:
Generally it's better to use `reverse_lazy` when instantiating URLs at the class level, because `reverse` executes immediately when the module is loaded and requires the URLConf to be loaded as well.

I think with tests it's generally not an issue because the URLConfig is already loaded by the time this module is loaded, but let's stick with using `reverse_lazy` at the class level to be safe.

```suggestion
    path = reverse_lazy("contact-list")
```